deployment/streaming/Makefile

```diff
@@ -34,7 +48,19 @@ endif
 
 .PHONY: build-container login-aws-registry tag-image push-image
 
-build-container:
+APPS_SRC_DIR := ../../src/apps
+APPS_ASSEMBLY := ${APPS_SRC_DIR}/target/scala-2.11/${OSMESA_APPS_JAR}
+
+rwildcard=$(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2) $(filter $(subst *,%,$2),$d))
```

Comment:
👀 what does this do?

Reply:
Neat trick to allow us to check for changes to `.scala` files so that we don't needlessly rebuild the assembly.  This gets used like a function (the $1 and $2 are the arguments).  See the invocation in the rule just below.  Thanks to smarter people than me for figuring this out.